pkg/modules/reverseproxy.go

```
@@ -295,11 +322,151 @@ func ReverseProxyInjectDeployment(dp v1.DeploymentApplyConfiguration, cr csmv1.C
 		}
 	}
 
+	// Dynamic secret/configMap mounting is only supported in v2.14.0 and above
+	secretSupported, _ := utils.MinVersionCheck(drivers.PowerMaxMountCredentialMinVersion, cr.Spec.Driver.ConfigVersion)
```

ignoring error values